.docker/docker-compose.yml

```diff
@@ -15,3 +17,11 @@ services:
       - '8081:8000'
     depends_on:
       - app
+  db:
+    image: postgres:12.1-alpine
```

Comment:
PostgreSQL 12 is not generally available yet on cloud providers such as [AWS RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.DBVersions); stick to `11.5` at most